icd/api/pipeline_binary_cache.cpp

```diff
@@ -37,6 +37,9 @@
 #include "palVectorImpl.h"
 #include "palHashMapImpl.h"
 #include "palFile.h"
+
+#include "include/binary_cache_serialization.h"
```

Comment:
Can you put this up with the include of pipeline_binary_cache.h so the xgl headers are together?